### PR TITLE
Fix broken Carthage support

### PR DESCRIPTION
**Pull request by tomco**
It was broken since there was no dependency defined towards *Swift-PureJsonSerializer*.

### CODE REVIEW

File: Genome/Source/Foundation/Foundation.swift [hunk 1]
```diff
@@ -8,6 +8,7 @@
 //  MIT
 
 import PureJsonSerializer
+import Foundation
```
Comment (tomco) [inline Genome/Source/Foundation/Foundation.swift]: Did no research why, but without this import the OS X framework will fail during linking

Comment (LoganWright) [inline Genome/Source/Foundation/Foundation.swift]: I think they changed something to do with the default import of Foundation when dealing with some frameworks.  Probably a good addition going forward.

**Comment by LoganWright** [comment]
@tomco 

Thanks for adding this, I'm not a carthage user personally so I rely on these types of additions for support!

Once the json fork gets merged, we'll update the dependencies to point to that branch and update.

**Comment by LoganWright** [comment]
@tomco PureSwiftJson should be good to go, would you mind update the urls to that repo, then let's get this merged in.

Thanks again!

**Comment by tomco** [comment]
@LoganWright all done. ready to get merged from my side

**Comment by LoganWright** [comment]
@tomco Thanks for the additions, just pushed a new tag, should be good to go.

**Comment by tomco** [comment]
@LoganWright great. I will check tomorrow and make sure everything works.

**Pull request merged**